spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/autoconfig/InfraConfigurationTest.java

```diff
@@ -218,35 +218,36 @@ public CountDownLatch eventHandlerInvocations() {
         }
 
         @Bean
-        public Set<String> handlingOutcome() {
-            return spy(new HashSet<>());
+        public Queue<String> handlingOutcome() {
+            return new PriorityQueue<>();
```

Comment:
Shouldn't this be a simple FiFo Queue, such as `LinkedList`?

Reply:
You're right that makes more sense. Will adjust.